src/DssSpell.sol

```diff
@@ -71,12 +44,19 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     //
 
     // --- Rates ---
-    //uint256 constant FOUR_FIVE_PCT_RATE      = 1000000001395766281313196627;
+    // uint256 constant FOUR_FIVE_PCT_RATE      = 1000000001395766281313196627;
+
+    // Turn office hours off
+    function officeHours() public override returns (bool) {
+        return true;
```

Comment:
office hours are on by default by way of the execlib. We can probably just comment this function out or remove it entirely because it doesn't match the comment here.